src/webrtc.ts

```diff
@@ -613,7 +614,11 @@ export class WebrtcExtension extends EventEmitter implements StreamingClientExte
    * Stanza Handlers
    */
   private async handlePropose (msg: ProposeStanza) {
-    if (msg.from === this.jid) {
+    // if (msg.from === this.jid) {
+    //   return;
+    // }
```

Comment:
We had this condition originally... It's currently commented out because I'm actually not sure if it is needed or not with the change we are planning on making.  I spoke with Jon and we sorta got to the point of "well it probably doesn't _hurt_" so definitely open to suggestions and input here.